client/optimise.py

```diff
@@ -53,6 +53,9 @@ def delete_job(self, world_id: str, job: str, reason: str) -> requests.Response:
     def get_jobs(self, world_id: str) -> GetJobsResponse:
         return self._get_list(f"worlds/{world_id}/jobs?pageSize=1000")
 
+    def get_jobs_filtered(self, world_id: str) -> GetJobsResponse:
```

Comment:
This looks like it's identical to `get_jobs` above? It also appears unused?

Reply:
Removed in https://github.com/ONSdigital/blaise-totalmobile-services/pull/194